homeassistant/components/music_assistant/services.yaml

```diff
@@ -0,0 +1,145 @@
+play_media:
+  target:
+    entity:
+      domain: media_player
+      integration: music_assistant
+      supported_features:
+        - media_player.MediaPlayerEntityFeature.PLAY_MEDIA
+  fields:
+    media_id:
+      required: true
+      example: "spotify://playlist/aabbccddeeff"
+      selector:
+        object:
+    media_type:
+      required: false
+      example: "playlist"
+      selector:
+        select:
+          options:
+            - artist
+            - album
+            - playlist
+            - track
+            - radio
```

Comment:
Please add a translation key to this selector so we can translate the possible values